File: solved/boy_or_girl.cpp
#include <iostream>
#include <set>

//link: https://vjudge.net/problem/CodeForces-236A

using namespace std;

int main()
{
    set <char> data;
    string input;

    cin >> input;

    for(int i=0; input[i] != '\0'; i++)
        data.insert(input[i]);
    if(data.size() % 2 == 0)
        cout << "CHAT WITH HER!" << endl;
    else
        cout << "IGNORE HIM!" << endl;
}

File: codeforces/Boy-or-Girl.java
import java.util.*;
// link: https://vjudge.net/problem/CodeForces-236A/origin
public class Solution{
    public static void main(String[] ags){
        Scanner sc = new Scanner(System.in);
        Set<Character> data = new HashSet<Character>();

        String input = sc.next();
        char[] array = input.toCharArray();

        for(int i=0; i<array.length; i++)
            data.add(array[i]);
        if(data.size() % 2 == 0)
            System.out.println("CHAT WITH HER!");
        else
            System.out.println("IGNORE HIM!");
    }
}
